collect_app/src/main/java/org/odk/collect/android/activities/AppListActivity.java

```diff
@@ -214,8 +214,7 @@ public boolean onMenuItemActionCollapse(MenuItem item) {
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        int i = item.getItemId();
-        if (i == R.id.menu_sort) {
+        if (item.getItemId() == R.id.menu_sort) {
```

Comment:
Did you use IDEA’s Refactor -> Inline for this? If so, it requires no review.

Reply:
No the last commit as I said I made manually, but it's pretty small.